src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-aumbry.py

```diff
@@ -0,0 +1,17 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2020 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+# Tested with Aumbry 0.10.0
+
+from PyInstaller.utils.hooks import collect_all
+
+datas, binaries, hiddenimports = collect_all('aumbry')
```

Comment:
`collect_all()` is incredibly inefficient. Does it really need all of data files, binaries, hidden imports, metadata and dependencies? I see no data files or binaries in the aumbry package. The only thing I see in there is [use of importlib](https://github.com/pyarmory/aumbry/blob/b1f8c4dee7a1f202c6266d56056a517085f9b5e8/aumbry/contract.py#L22) which will require `collect_submodules('aumbry')`.

Reply:
Ok

I am not sure right now, I will check it again!
Thank you for your advice.